dashboard/test/models/pd/enrollment_test.rb

```diff
@@ -134,4 +136,36 @@ class Pd::EnrollmentTest < ActiveSupport::TestCase
     enrollment.send_exit_survey
     assert_not_nil enrollment.reload.survey_sent_at
   end
+
+  test 'name is deprecated and calls through to full_name' do
+    enrollment = create :pd_enrollment
+    enrollment.expects(:full_name)
+    assert_deprecated 'name is deprecated. Use first_name & last_name instead.' do
+      enrollment.name
+    end
+
+    enrollment.expects('full_name=' => 'First Last')
+    assert_deprecated 'name is deprecated. Use first_name & last_name instead.' do
+      enrollment.name = 'First Last'
+    end
```

Comment:
I believe it is pure Mocha, but I can't find this particular syntax in their docs now. The docs show this syntax for expects: `enrollment.expects('full_name=').with('First Last')`, but I like the concise hash syntax. Oddly they do show the hash syntax for `stub`, just not `expects`.